modules/oceanbase-ce/src/main/java/org/testcontainers/containers/OceanBaseContainer.java

```diff
@@ -0,0 +1,101 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+
+/**
+ * Testcontainers implementation for OceanBase.
+ * <p>
+ * Supported image: {@code oceanbase/oceanbase-ce}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>SQL: 2881</li>
+ *     <li>RPC: 2882</li>
+ * </ul>
+ */
+public class OceanBaseContainer extends JdbcDatabaseContainer<OceanBaseContainer> {
+
+    static final String NAME = "oceanbase";
+
+    static final String DOCKER_IMAGE_NAME = "oceanbase/oceanbase-ce";
+
+    static final String DEFAULT_TAG = "4.1.0.0";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse(DOCKER_IMAGE_NAME);
+
+    private static final Integer SQL_PORT = 2881;
+
+    private static final Integer RPC_PORT = 2882;
+
+    private final String databaseName;
+
+    private final String username;
+
+    private final String password;
+
+    public OceanBaseContainer(String dockerImageName) {
+        this(DockerImageName.parse(dockerImageName));
+    }
+
+    public OceanBaseContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        this.databaseName = "test";
+        this.username = "root@test";
+        this.password = "";
+
+        this.waitStrategy = Wait.forLogMessage(".*boot success!.*", 1).withStartupTimeout(Duration.ofMinutes(5));
```

Comment:
5 minutes? 👀 it really takes that long?

Reply:
The process of downloading images, dependencies, and starting is usually completed within 2 minutes in our development environment. I have given a large value here, and I think it can be changed to 3 minutes.

Reply:
the default in Testcontainers implementations is 1 min at max. This is only for service startup, pulling and container creation is not considered. Is there any way to improve startup? some flags that can help?

Reply:
> the default in Testcontainers implementations is 1 min at max. This is only for service startup, pulling and container creation is not considered. Is there any way to improve startup? some flags that can help?

In order to prevent the image from being too large, we put the installation process into the startup process of container, so it is always necessary to obtain dependencies through the network when starting the container for the first time.